metrics/web/lib/dashboard/presentation/state/project_metrics_notifier.dart

```diff
@@ -567,6 +596,33 @@ class ProjectMetricsNotifier extends ChangeNotifier {
     }
   }
 
+  /// Updates the [_currentPageParameters] with the given [projectFilter] and
+  /// [projectGroupId].
+  ///
+  /// If the given [projectFilter] or [projectGroupId] is `null`,
+  /// current values are used.
+  void _updateCurrentPageParameters({
+    String projectFilter,
+    String projectGroupId,
```

Comment:
🔨
It seems like there is no need for parameters here since we want to update the page parameters only to current values, so we can use them directly. What do you think?